install-sdk-packages.ps1

```diff
@@ -0,0 +1,61 @@
+#########################################################
+#                                                       #
+#  Install necessary Android packages to run project	#
+#                                                       #
+#########################################################
+
+""
+
+# Prompt package download
+Write-Host "Are you sure you want to install ALL needed packages for this project?"
+$sure = Read-Host -Prompt "This may take awhile....		[Y/n]"
+""
+
+# Check for user response
+$sure=$sure -replace "^[Yy]$", 'y' 
+if($sure -ne "y")
+{
+	exit 0
+}
+
+# Check for SDK Home
+if(-NOT (Test-Path -path $env:ANDROID_HOME))
+{
+	Write-Host "ANDROID_HOME not defined" -Foreground Red
+	""
+	Write-Host "Assuming SDK is at $env:LOCALAPPDATA\Android\sdk" -Foreground Yellow
+	""
+	[Environment]::SetEnvironmentVariable("ANDROID_HOME", "$env:LOCALAPPDATA\Android\sdk", "Machine")
```

Comment:
Is this ok at just `"Machine"`? I see `[EnvironmentVariableTarget]::Machine` other places

Reply:
Yea it's one or the other...I kinda did both didn't I😂😂